Dockerfile

```diff
@@ -49,7 +49,11 @@ ADD .htmltest.yml .htmltest.yml
 RUN htmltest
 
 FROM build-base as update-modules
-ARG MODULE="-u"
+ARG MODULE
+RUN if [ -z "$MODULE" ]; then \
+      >&2 echo "MODULE unset"; \
+      exit 1; \
+    fi
```

Comment:
`-u` was never really a useful default so instead I'm opting to fail if MODULE is unset